static/images/Image-Licensing-and-Technical-Notes.txt

```diff
@@ -19,11 +19,16 @@ The license terms above apply to the following images:
     leftrightarrow.svg        U+2194             KaTeX Main
     leftrightharpoons.svg     U+21BC/21B1        KaTeX Main
     mapsto.svg                U+21A6             KaTeX Main
-    overbrace.svg             U+23A9/23A8/23A7   KaTeX Size 4 Regular
+    overbrace.svg             U+23A9/23A8/23A7   KaTeX_Size4-Regular
```

Comment:
Yeah, I was writing font names for the surds and I changed this to be consistent.